src/main/java/kitchenpos/products/tobe/repository/ProductRepository.java

```diff
@@ -0,0 +1,8 @@
+package kitchenpos.products.tobe.repository;
+
+import kitchenpos.products.tobe.domain.Product;
+import org.springframework.data.jpa.repository.JpaRepository;
+
+import java.util.UUID;
+
+public interface ProductRepository extends JpaRepository<Product, UUID> {}
```

Comment:
기존 domain에 있던 ProductRepository를 그대로 가져 오지 않고 변형 하신 이유가 있으실까요?
```suggestion
public interface ProductRepository {
    Product save(Product product);

    Optional<Product> findById(UUID id);

    List<Product> findAll();

    List<Product> findAllByIdIn(List<UUID> ids);
}
```

Reply:
옮기는 과정에 무언가 착오가 있던 모양이었나봅니다..!!